contrib/datawave-quickstart/bin/services/accumulo/bootstrap.sh

```diff
@@ -274,15 +283,6 @@ function accumuloPrintenv() {
    echo
 }
 
-function accumuloPidList() {
-   # Refresh pid lists
-   accumuloIsRunning
-   zookeeperIsRunning
-   if [[ -n "${DW_ACCUMULO_PID_LIST}" || -n "${DW_ZOOKEEPER_PID_LIST}" ]] ; then
-      echo "${DW_ACCUMULO_PID_LIST} ${DW_ZOOKEEPER_PID_LIST}"
-   fi
-}
```

Comment:
This removal, combined with the other changes here, breaks ```allStop --hard``` [here](https://github.com/NationalSecurityAgency/datawave/blob/integration/contrib/datawave-quickstart/bin/common.sh#L163)